README.md

```diff
@@ -72,7 +72,13 @@ jenkins/track-upstream-and-package.pl syntax OK
 # jenkins jobs
 There are manually maintained jobs and some jobs are now using
 [jenkins-job-builder](http://docs.openstack.org/infra/jenkins-job-builder/)
-which defines jobs in yaml format.
+which defines jobs in yaml format. New jobs should always be defined
+in yaml format.
 To update jobs which land on ci.opensuse.org, run:
 
-   jenkins-jobs --ignore-cache update scripts/jenkins/jobs-obs/
+    jenkins-jobs --ignore-cache update scripts/jenkins/jobs-obs/
+
+To update jobs which land on the SUSE internal CI, run:
+
+    jenkins-jobs --ignore-cache update \
+        scripts/jenkins/jobs-ibs/:scripts/jenkins/jobs-ibs/templates/
```

Comment:
The commands fail for me:  
```
jenkins_jobs.errors.JenkinsJobsException: A valid configuration file is required when not run as a test
/etc/jenkins_jobs/jenkins_jobs.ini is not a valid .ini file
```
Shouldn't we reference the config file (I assume scripts/jenkins/jenkins_jobs.ini.sample is the template for that) somewhere here?